libsolidity/codegen/ir/IRGeneratorForStatements.cpp

```diff
@@ -1212,7 +1212,20 @@ void IRGeneratorForStatements::endVisit(MemberAccess const& _memberAccess)
 			solAssert(false, "Blockhash has been removed.");
 		else if (member == "creationCode" || member == "runtimeCode")
 		{
-			solUnimplementedAssert(false, "");
+			solUnimplementedAssert(member != "runtimeCode", "");
```

Comment:
Why not also support the `runtimeCode`?

Does it require more than `runtimeObjectName(contract)` as long as we assert immutables are not used?

Reply:
Yep - I tried that, but the runtime object is a nested yul object inside the creation yul object and apparently we currently have scoping rules for yul objects that prevent you from accessing the inner yul object from a different yul object, so for supporting runtime objects (at least) that would need to change.

Reply:
You mean #6739?

Reply:
I didn't know there was an issue for that :-). But yeah, that one exactly :-).